eip-registry.md

```diff
@@ -16,15 +16,15 @@ This specification defines a standardized way of storing and retrieving an entit
 
 ## Motivation
 
-The standardization of non-interactive stealth address generation holds the potential to greatly enhance the privacy capabilities of Ethereum by enabling the recipient of a transfer to remain anonymous when receiving an asset. Making stealth addresses easy to use by providing a shared, standardized registry for make it even easier to non-interactively send and receive assets with stealth addresses.
+The standardization of  stealth address generation holds the potential to greatly enhance the privacy capabilities of Ethereum by enabling the recipient of a transfer to remain anonymous when receiving an asset. By introducing a central place for users to store their stealth meta-addresses EOAs and contracts can engage in programmatic ways, guranteeing interoperability accross a specific chain, while offering enough flexibilty to implement improved stealth address schemes in the future.
```

Comment:
A few typos, and some more material changes. Feel free to pushback on whatever parts you want
```suggestion
The standardization of stealth address generation holds the potential to greatly enhance the privacy capabilities of Ethereum by enabling the recipient of a transfer to remain anonymous when receiving an asset. By introducing a central smart contract for users to store their stealth meta-addresseses, EOAs and contracts can programmatically engage in stealth interactions using a variety of stealth address scehemes.
```